src/pages/AddHotel/AddHotelForm.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint linebreak-style: ["error", "windows"] */
+// import { IKContext, IKUpload } from "imagekitio-react";
```

Comment:
- [ ] Kindly remove this old piece of commented code to improve the repo's cleanliness. 👍